src/main/java/AttendanceTracker/App.java

```diff
@@ -2,44 +2,14 @@
  * This Java source file was generated by the Gradle 'init' task.
  */
 package AttendanceTracker;
-import java.util.Scanner;
-import java.time.*;
+import static AttendanceTracker.UserChooseYear.userChooseYearStyle;
```

Comment:
importing isn't necessary with packages just reference the class using class.method() in the code